### DISCUSSION TITLE
想问一下开发大佬 rk3328要用哪个升级文件

### DISCUSSION THREAD

**Discussion by wjstyle** [messense/aliyundrive-webdav#475] [Q&A]
https://github.com/messense/aliyundrive-webdav/releases/tag/v1.5.1
想问一下我要升级aliyundrive-webdav-1.5.1
不知道要下载哪个。。。
小白问题 希望开发大佬答复

**Comment by 0x676e67** [comment]
> https://github.com/messense/aliyundrive-webdav/releases/tag/v1.5.1
> 想问一下我要升级aliyundrive-webdav-1.5.1
> 不知道要下载哪个。。。
> 小白问题 希望开发大佬答复

a53后缀
